rmf_robot_sim_gz_classic_plugins/CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ find_package(rmf_fleet_msgs REQUIRED)
 find_package(rmf_building_map_msgs REQUIRED)
 find_package(rmf_robot_sim_common REQUIRED)
 
+find_package(ignition-cmake2 REQUIRED)
```

Comment:
No Idea why I need this here but oh-well. I guess I can move the sdf code into the header and then not need it?